<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QLabel" name="labelPRUEBA">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="ImageView" name="imv1"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="separator"/>
    <addaction name="saveImgAction"/>
    <addaction name="separator"/>
    <addaction name="LoadImageAction"/>
   </widget>
   <widget class="QMenu" name="menuSegmentation">
    <property name="title">
     <string>Segmentation</string>
    </property>
    <addaction name="findRoiAction"/>
    <addaction name="actionContinueSeg"/>
   </widget>
   <widget class="QMenu" name="menuNetwork_Analysis">
    <property name="title">
     <string>Network Analysis</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSegmentation"/>
   <addaction name="menuNetwork_Analysis"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReset"/>
  </widget>
  <action name="openAction">
   <property name="text">
    <string>Open Stack</string>
   </property>
  </action>
  <action name="saveImgAction">
   <property name="text">
    <string>Save Binary Image</string>
   </property>
  </action>
  <action name="LoadImageAction">
   <property name="text">
    <string>Load Binary Image</string>
   </property>
  </action>
  <action name="findRoiAction">
   <property name="text">
    <string>Cell Segmentation</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionContinueSeg">
   <property name="text">
    <string>Load Segmentation</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
